pages/editor.vue

```diff
@@ -1,11 +1,14 @@
 <script setup>
     import ComicPanels from '~/components/ComicPanels.vue';
+    import iconConfig from '../config/iconsConfig';
 
     let layersShow = ref(false);
     let previewShow = ref(false);
     let catalogShow = ref(false);
+    let goingBackPopUpShow = ref(false);
     let selectedElementId = ref(null);
     let lockAspectRatio = ref(false);
+    let editor = ref(null);
```

Comment:
This ref isn't use right?

Reply:
It will be when we will change the pop-up on desktop